src/models/reposyncstate.ts

```diff
@@ -310,6 +310,32 @@ export class RepoSyncState extends Model implements RepoSyncStateProperties {
 		return repositories as RepoSyncState[];
 	}
 
+	static async findOneForRepoUrlAndRepoIdAndJiraHost(repoUrl: string, repoId: number, jiraHost: string):Promise<RepoSyncState | null> {
+		const results = await this.sequelize!.query(
+			`SELECT rss.* 
+			FROM "RepoSyncStates" rss
+			JOIN "Subscriptions" s ON rss."subscriptionId" = s."id"
```

Comment:
I have hesitation on the performance here:
1. Why do we need to use `JOIN`, can't we grab the subscriptionId first, and just search on RepoSyncStates table?
2. Why do we need to use LIKE? Can't we just do "repoUrl" = :repoUrl?

Not that it doesn't work, it is more like a performance concern.